test/Microsoft.Azure.WebJobs.EventHubs.UnitTests/EventHubTests.cs

```diff
@@ -228,9 +228,9 @@ public void InitializeFromHostMetadata()
                 {
                     c.AddInMemoryCollection(new Dictionary<string, string>
                     {
-                        { "EventHub:MaxBatchSize", "100" },
-                        { "EventHub:PrefetchCount", "200" },
-                        { "EventHub:BatchCheckpointFrequency", "5" },
+                        { "AzureWebJobs:extensions:EventHub:MaxBatchSize", "100" },
+                        { "AzureWebJobs:extensions:EventHub:PrefetchCount", "200" },
+                        { "AzureWebJobs:extensions:EventHub:BatchCheckpointFrequency", "5" },
```

Comment:
Just noticed this -- should this be `EventHubs` (plural)?

Reply:
It always gets me too, but I verified as I was doing this as every type is `EventHub...` so I kept it consistent here.